ansible/sample_production/teardown_workloads.yml

```
@@ -158,22 +158,6 @@
         - item.placement_group not in pg_undeletable
         - item.tenant_space not in ts_undeletable
 
-    - name: Remove HAP from volume(s)
-      purestorage.fusion.fusion_volume:
-        app_id: "{{ api_client }}"
-        key_file: "{{ priv_key_file }}"
-        name: "{{ item.name }}"
-        display_name: "{{ item.display_name }}"
-        size: "{{ item.size }}"
-        storage_class: "{{ item.storage_class }}"
-        tenant: "{{ item.tenant }}"
-        tenant_space: "{{ item.tenant_space }}"
-        placement_group: "{{ item.placement_group }}"
-        host_access_policies: []
-        state: "present"
-      with_items: "{{ volumes }}"
-      when: item.mount_path not in failed_unmount
-
     - name: Delete volume(s)
```

```suggestion
    - name: Eradicate volume(s)
```
I'd change this line just to emphasize it eradicates the volume.